.github/workflows/continous-integration.yml

```diff
@@ -16,9 +22,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: [ '7.3', '7.4', '8.0' ]
+        php: ["7.3", "7.4", "8.0", "8.1", "8.2", "8.3"]
```

Comment:
Do we _need_ to keep support for `7.3`, `7.4`, `8.0`? They're past EOL: https://www.php.net/supported-versions.php

Reply:
We don't, you have my permission to change it to "8.1", "8.2", and "8.3"